backend/api/helper/userpermission.py

```diff
@@ -0,0 +1,11 @@
+def is_permitted(roles):
```

Comment:
this method also needs the current user and the instruction it should compare the eligiblility to execute on